learnable/src/pages/WorkshopDetailPage.jsx

```diff
@@ -19,9 +15,7 @@ function WorkshopDetail(){
             setProjectData(data)
         })
     }, []);
-    console.log(projectData);
-
-
+    // console.log(projectData);
```

Comment:
```suggestion
```

Reply:
Same as above - hit commit suggestion button and git pull on your machine, or go into VS code and make the changes yourself and push them.